src/cpu/cpu_convolution_list.cpp

```diff
@@ -228,6 +228,7 @@ const std::map<conv_impl_key_t, std::vector<pd_create_f>> impl_list_map {
         CPU_INSTANCE_X64(jit_uni_x8s8s32x_convolution_fwd_t<avx2, s8, s8>)
         CPU_INSTANCE_X64(jit_uni_x8s8s32x_1x1_convolution_fwd_t<sse41, s8, s8>)
         CPU_INSTANCE_X64(jit_uni_x8s8s32x_convolution_fwd_t<sse41, s8, s8>)
+        CPU_INSTANCE_AARCH64_ACL(acl_gemm_convolution_fwd_t<s8, s8, s8, s32>)
```

Comment:
Just want to point out that this functionality is not covered by public CI. Similar to #886 enabling benchdnn in public CI is out of scope of this PR.

Reply:
Noted. We are planning an Aarch64-specific testing suite to cover most of our newly added functionality.